src/content/docs/ko/guides/middleware.mdx

```diff
@@ -196,6 +197,79 @@ auth 응답
 validation 응답
 ```
 
+## 리라이팅
+
+<p><Since v="4.13.0" /></p>
+
+`APIContext`는 `rewrite()`라는 메서드를 노출하는데, 이는 [Astro.rewrite](/ko/guides/routing/#리라이트)와 동일한 방식으로 작동합니다.
+
+방문자를 새 페이지로 [리디렉션](/ko/guides/routing/#동적-리디렉션)하지 않고 다른 페이지의 콘텐츠를 표시하려면 미들웨어에서 `context.rewrite()`를 사용하세요. 이렇게 하면 새 렌더링 단계가 트리거되어 모든 미들웨어가 다시 실행됩니다.
+
+```js title="src/middleware.js"
+import { isLoggedIn } from "~/auth.js"
+export function onRequest (context, next) {
+  if (!isLoggedIn(context) {
```

Comment:
```suggestion
  if (!isLoggedIn(context)) {
```

I fixed this in https://github.com/withastro/docs/pull/8994

Reply:
Thanks!